src/main/MQ2ImGuiConsole.cpp

```diff
@@ -1328,6 +1354,41 @@ static void MakeColorGradient(float frequency1, float frequency2, float frequenc
 }
 
 //----------------------------------------------------------------------------
+void AlphaSetting(bool AlphaBool)
+{
+	if (AlphaBool || !s_alphaBool || s_mouseHoveringConsoleTimer != 0)
+	{
+		ImGui::SetNextWindowBgAlpha(1.0f);
+	}
+	else 
+	{
+		ImGui::SetNextWindowBgAlpha(.1f);
+	}
+}
+
+bool MouseOverUpdate()
+{
+	if (ImGui::IsWindowHovered())
+	{
+		// This may not be the MQ way of doing it Its just a timer for Mouse over window and drop down but makes it very slick giving a delay to fade
+		// This Actually Covers up another issue with Button Presses as with a button down you are not hoveringoverwindow
+		s_mouseHoveringConsoleTimer = 1200;
+		return true;
+	}
+	else
+	{
+		if (s_mouseHoveringConsoleTimer > 0)
+		{
+			s_mouseHoveringConsoleTimer = s_mouseHoveringConsoleTimer - 1;
```

Comment:
This logic is frame rate dependent, and will vary heavily based on how often this function updates.

This needs to be changes to be framerate independent. Look at using std::chrono::stready_clock and measure the elapsed time instead of decrementing a counter.